main.go

```diff
@@ -92,6 +91,10 @@ func main() {
 	var retryInterval int
 	var printVersion bool
 
+	// Setup the interrupt handler to gracefully exit.
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, os.Interrupt)
```

Comment:
I think we should handle more than just `SIGINT` here. Instead of `os` package (which does not have alias for `SIGTERM`), you could use signals from `syscall`:
```go
signal.Notify(c, syscall.SIGINT, syscall.SIGTERM)
```

I know that the old code just handled `os.Interrupt` but it would be nice to do it properly this time.

Reply:
I was curious whether to add SIGTERM, as it isn’t supported in Windows.
But I checked nats-server.
We can capture both signals.